pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go

```diff
@@ -170,6 +173,21 @@ type SpiderOvsCniConfig struct {
 	SpiderpoolConfigPools *SpiderpoolPools `json:"ippools,omitempty"`
 }
 
+type SpiderHostDeviceCniConfig struct {
+	// +kubebuilder:validation:Optional
+	Device string `json:"deviceName,omitempty"`
+	// +kubebuilder:validation:Optional
+	HWAddr string `json:"hwAddr,omitempty"`
+	// +kubebuilder:validation:Optional
+	DPDKMode bool `json:"enableDPDK,omitempty"`
```

Comment:
I think we don't need `DPDKMode`, this field can't be configured. The host-device plugin can detect it is in DPDK mode automatically by checking the Device's driver. see https://github.com/containernetworking/plugins/blob/c860b78de419336c418fee93fc5cf0f6c9b115ba/plugins/main/host-device/host-device.go#L113.